content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: blog
+title: "Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security."
+date: 2023-01-17
+slug: "confidential-kubernetes"
+---
+
+**Authors:** Fabian Kammel (Edgeless Systems), (more to add...)
+
+
+# Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security.
+
+In this blog post, we will introduce the concept of Confidential Computing (CC) to improve the security and privacy properties of any computing environment. Further, we will show how the Cloud-Native ecosystem, and Kubernetes in particular, can benefit from the new compute paradigm.
+
+## Intro
+
+The [Whitepaper](https://confidentialcomputing.io/wp-content/uploads/sites/85/2022/11/CCC-A-Technical-Analysis-of-Confidential-Computing-v1.2_updated_2022-11-02.pdf) produced by the [Confidential Computing Consortium](https://confidentialcomputing.io/), a [LinuxFoundation](https://confidentialcomputing.io/wp-content/uploads/sites/85/2019/12/CCC_Overview.pdf) project, provides a great motivation for the usage of Confidential Computing:
```

Comment:
- “Linux Foundation” (with a space)
- To hyperlink to https://confidentialcomputing.io/wp-content/uploads/sites/85/2019/12/CCC_Overview.pdf, use the link text “Defining and Enabling Confidential Computing” (you will need to reword)